scipy/special/add_newdocs.py

```diff
@@ -3035,6 +3035,52 @@ def add_newdoc(place, name, doc):
     hyp2f1(a, b, c, z)
 
     Gauss hypergeometric function 2F1(a, b; c; z).
+
+    Parameters
+    ----------
+    a : double argument.
+
+    b : double argument.
+
+    c : double argument.
+
+    z : double argument.
+
+    Returns
+    -------
+    y : double argument
+        y being the solution to the second-order linear ordinary differential equation.
```

Comment:
You haven't told the user that this function satisfies a second-order linear equation, so this isn't a good way to go. Just say something like "values of the Gaussian hypergeometric function".